### DISCUSSION TITLE
Documentation for open source project (ohkrab.dev)

### DISCUSSION THREAD

**Discussion by qbart** [thuliteio/doks#1002] [Showcase]
Hello, [here](https://ohkrab.dev/) is the doks powered documentation for the open source project I am working on.

**Comment by rachfop** [comment]
How did you add the styling components to your table of contents? I like the SVGs there.

**Reply by qbart** [reply]
I've added `.Resources.Get` to [menu](https://github.com/ohkrab/docs/blob/9b9e256286ad89c6fda38d1e02ebb90efa0c7232/layouts/partials/sidebar/auto-default-menu.html#L8),
so if my content section contains `icon.svg` like [here](https://github.com/ohkrab/docs/tree/9b9e256286ad89c6fda38d1e02ebb90efa0c7232/content/en/docs/functions) or [here](https://github.com/ohkrab/docs/tree/9b9e256286ad89c6fda38d1e02ebb90efa0c7232/content/en/docs/commands) it will display an icon.
